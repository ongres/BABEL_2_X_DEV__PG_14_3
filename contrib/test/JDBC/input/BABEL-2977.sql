create login alreadyexists with password='12345678';
GO

create login alreadyexists with password='12345678';
GO

drop login nosuchlogin;
GO

alter login nosuchlogin with default_database=mydb;
GO

DROP login alreadyexists;
GO
