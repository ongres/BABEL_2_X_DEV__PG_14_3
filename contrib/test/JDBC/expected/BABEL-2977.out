create login alreadyexists with password='12345678';
GO

create login alreadyexists with password='12345678';
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: The Server principal 'alreadyexists' already exists)~~


drop login nosuchlogin;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: Cannot drop the login 'nosuchlogin', because it does not exist or you do not have permission.)~~


alter login nosuchlogin with default_database=mydb;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: Cannot drop the login 'nosuchlogin', because it does not exist or you do not have permission.)~~


DROP login alreadyexists;
GO
